examples/caffeine/src/main/java/com/giffing/bucket4j/spring/boot/starter/examples/caffeine/TestController.java

```diff
@@ -1,20 +1,146 @@
 package com.giffing.bucket4j.spring.boot.starter.examples.caffeine;
 
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Valid;
+import jakarta.validation.Validator;
+
+import org.springframework.context.support.DefaultMessageSourceResolvable;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import com.giffing.bucket4j.spring.boot.starter.config.cache.CacheManager;
+import com.giffing.bucket4j.spring.boot.starter.context.properties.BandWidth;
+import com.giffing.bucket4j.spring.boot.starter.context.properties.Bucket4JConfiguration;
+import com.giffing.bucket4j.spring.boot.starter.context.properties.RateLimit;
 
 @RestController
 public class TestController {
-	
+
+	private final Validator validator;
+	private final CacheManager<String, Bucket4JConfiguration> configCacheManager;
+
+	public TestController(Validator validator, CacheManager<String, Bucket4JConfiguration> configCacheManager) {
+		this.validator = validator;
+		this.configCacheManager = configCacheManager;
+	}
+
+	@GetMapping("unsecure")
+	public ResponseEntity unsecure() {
+		return ResponseEntity.ok().build();
+	}
+
 	@GetMapping("hello")
 	public ResponseEntity<String> hello() {
 		return ResponseEntity.ok("Hello World");
 	}
-	
+
 	@GetMapping("world")
 	public ResponseEntity<String> world() {
 		return ResponseEntity.ok("Hello World");
 	}
-	
+
+	@PostMapping("filters/{filterId}")
+	public ResponseEntity updateConfig(
```

Comment:
I havn't fully reviewed the PR but can you move this configuration to the bucket4j-spring-boot-starter project? So the update mechanism is available for the users wihtout the necessity to know the required validation.

Reply:
what do you mean with moving the configuration?

Reply:
Do you mean something like this?
https://github.com/Edwin9292/bucket4j-spring-boot-starter/pull/3 

Since the cacheManager is generic it's not possible to add such specific checks in that class. For this reason I made a Utils class for checking if the update is valid. The Jakarta validation should in my opinion still be done in the controller, same as the getting/updating of the cache. 

Please let me know if this pull request would resolve your comment or if you would like to see additional changes.

Reply:
https://github.com/Edwin9292/bucket4j-spring-boot-starter/pull/3 is good. So the user must not implement the logic himself. Especially if there are changes in the logic. 

But I mean to provide a general update Controller out of the box. On solution would be a custom Spring Actuator Endpoint. It's only a proposal. It's ok as it is.
https://www.baeldung.com/spring-boot-actuators#custom-endpoint

Reply:
Never worked with Spring Actuator before, but the example looks like it's an easy way to create endpoints as Beans. I can imagine this is a nice solution for users that don't wish (or know how) to configure the endpoint themself. 

However, I do see potential risks when adding such an endpoint in the background. Especially when people don't know that they have to configure Spring Security for that endpoint, since it could allow anyone to update configurations. Even if the endpoint could be enabled/disabled by some application property, I'm not sure if the company i'm creating the feature for would accept it. They like to have full control over which endpoints they have and what behavior they contain.

Since you agree with the provided example I will merge that branch. 
In case I do have spare time for implementing the other option that i mentioned, I'll create another pull request and send you a link for reviewing.